Whatsapp.post() results in 401 error

First, thank you for the good and very promising library.

**Issue details**
- Calling Whatsapp.post() always results in 401 error.
- secret and token are checked.
- Webhook URL register was previouly successfuly done.
- Previous 0.8 version was working fine.
- JSON string (message data) sent by Meta/Whatsapp servers in post body seems ok (it is correctly displayed in console.log).
- Library version: 1.0.0-beta.5
- Node 18.15.0

**app.js**
```
import { createServer } from "http";
import crypto from "crypto";';
import { fetch } from "undici";
...

const Whatsapp = new WhatsAppAPI({ 
    token: MY_TOKEN, 
    appSecret: MY_APP_SECRET, 
    ponyfill: {
        fetch, 
        subtle: crypto.subtle
    }
});

const requestListener = function (req, res) {
  if (req.method == "POST") {
          let body = [];
          req.on('data', (chunk) => {
              body.push(chunk);
          }).on('end', () => {
              var bodyStr = Buffer.concat(body).toString();
              console.log(bodyStr);
              //Handlers.post(JSON.parse(bodyStr), onMessage);
              Whatsapp.post(JSON.parse(bodyStr), bodyStr, req.headers["X-Hub-Signature-256"]).then(()=>{
                  res.end(JSON.stringify({url: req.url, error: 0}));
              }).catch((err) => {
                   // Always catchs here (err = 401).
                   console.error(err);
               });
          });
   }
};

Whatsapp.on.message = async ({ phoneID, from, message, name, raw }) => {
   console.log(`User ${name} (${from}) sent to bot ${phoneID} ${JSON.stringify(message)}`);
};

const server = createServer(requestListener);
server.listen(80, () => {
    console.log(`Server is running`);
});
```

**package.json**
```
{
  "type": "module",
  "dependencies": {
    "crypto": "^1.0.1",
    "dotenv": "^16.0.3",
    "query-string": "^8.1.0",
    "undici": "^5.21.0",
    "whatsapp-api-js": "^1.0.0-beta.5"
  }
}
```


**Additional Suggestions**
I got difficulties to setup the new library version (1.0.0) compared to the previous version.

- First, the @latest version doest not corresponds to github readme file, which made me a  little confuse. This resulted in "Cannot find module 'whatsapp-api-js/messages'" and another errors. 
- After install the 1.0.0 beta version I got "crypto undefined" error when running my app. I had to look at the lib code to realize that it is necessary to provide a **ponyfill object (?) with "fetch" and "subtle" attributes (?)**. I had difficulties to understand and setup these parameters,  so please consider provide a more detailed documentation about how setup these parameters (maybe a full setup example using common libraries).

--- a/src/index.ts
+++ b/src/index.ts
@@ -579,13 +579,52 @@ export default class WhatsAppAPI {
      * POST helper, must be called inside the post function of your code.
      * When setting up the webhook, only subscribe to messages. Other subscritions support might be added later.
      *
+     * @example
+     * ```ts
+     * // Simple http example implementation with Whatsapp.post() on Node@^19
+     * import WhatsAppAPI from "whatsapp-api-js";
+     * import { NodeNext } from "whatsapp-api-js/setup/node";
+     * import { createServer } from "http";
+     *
+     * const token = "token";
+     * const appSecret = "appSecret";
+     * const Whatsapp = new WhatsAppAPI(NodeNext({ token, appSecret }));
+     *
+     * function handler(req, res) {
+     *     if (req.method == "POST") {
+     *         const chunks = [];
+     *         req.on("data", (chunk) => chunks.push(chunk));
+     *
+     *         req.on("end", async () => {
+     *             const body = Buffer.concat(chunks).toString();
+     *
+     *             try {
+     *                 const response = await Whatsapp.post(JSON.parse(body), body, req.headers["x-hub-signature-256"]);
+     *                 res.writeHead(response);
+     *             } catch (err) {
+     *                 res.writeHead(err);
+     *             }
+     *
+     *             res.end();
+     *         });
+     *     } else res.writeHead(501).end();
+     * };
+     *
+     * Whatsapp.on.message = ({ phoneID, from, message, name }) => {
+     *     console.log(`User ${name} (${from}) sent to bot ${phoneID} a(n) ${message.type}`);
+     * };
+     *
+     * const server = createServer(handler);
+     * server.listen(3000);
+     * ```
+     *
      * @param data - The POSTed data object sent by Whatsapp
      * @param raw_body - The raw body of the POST request
      * @param signature - The x-hub-signature-256 (all lowercase) header signature sent by Whatsapp
      * @returns 200, it's the expected http/s response code
      * @throws 500 if secure and the appSecret isn't specified
-     * @throws 501 if secure and crypto.subtle verify method or ponyfill isn't available
-     * @throws 400 if secure and the rawBody is missing
+     * @throws 501 if secure and crypto.subtle or ponyfill isn't available
+     * @throws 400 if secure and the raw body is missing
      * @throws 401 if secure and the signature is missing
      * @throws 401 if secure and the signature doesn't match the hash
      * @throws 400 if the POSTed data is not a valid Whatsapp API request

--- a/src/index.ts
+++ b/src/index.ts
@@ -581,7 +581,7 @@ export default class WhatsAppAPI {
      *
      * @param data - The POSTed data object sent by Whatsapp
      * @param raw_body - The raw body of the POST request
-     * @param signature - The X-Hub-Signature-256 header signature sent by Whatsapp
+     * @param signature - The x-hub-signature-256 (all lowercase) header signature sent by Whatsapp
      * @returns 200, it's the expected http/s response code
      * @throws 500 if secure and the appSecret isn't specified
      * @throws 501 if secure and crypto.subtle verify method or ponyfill isn't available

